torchrl/_utils.py

```diff
@@ -117,3 +118,24 @@ def prod(sequence):
         return math.prod(sequence)
     else:
         return int(np.prod(sequence))
+
+
+def get_binary_env_var(key):
```

Comment:
codecov complains about this being half tested, can we fix that?

Reply:
Added tests in order to cover every case.